Source/TelegAPI.Bot.Impl.pas

```diff
@@ -825,6 +813,31 @@ function TTelegramBot.ForwardMessage(ChatId, FromChatId: TValue; MessageId: Int6
   end;
 end;
 
+function TTelegramBot.GetArrayFromMethod<TI>(TgClass: TBaseJsonClass; const Method: string; Parameters: TDictionary<string, TValue>): TArray<TI>;
+var LJsonArr: TJSONArray;
+    I: Integer;
+    GUID: TGuid;
+begin
+  LJsonArr := GetJSONArrayFromMethod(Method, Parameters);
+
+  //cache value fot further use
+  GUID:=GetTypeData(TypeInfo(TI))^.Guid;
+
+  //check for TI interface support
+  if TgClass.GetInterfaceEntry(GUID)=nil then
+    raise Exception.Create('GetArrayFromMethod: unsupported interface for '+TgClass.ClassName);
+
+  try
```

Comment:
After try we can check count value and exit if no work to do:
if LJsonArr.Count=0 then exit